style.css

```diff
@@ -68,46 +65,333 @@
     line-height: 24px;
     color: #344563;
     margin-bottom: 10px;
-    padding-left: 20px;
   }
 
   .social-media {
     display: flex;
     align-items: center;
-    gap: 10px;
+    gap: 18px;
+    margin-bottom: 15px;
+  }
+
+  .link-text a {
+    text-decoration: none;
   }
 
   .social-media li {
     list-style: none;
-    padding-left: 20px;
   }
 
   .social-media li a {
     text-decoration: none;
     color: #505f79;
   }
 
+  .language-set a img {
+    padding-left: 20px;
+    margin-top: 10px;
+  }
+
   .social-media li a img {
     width: 20px;
   }
 
-  .btn {
+  .link-text {
     font-size: 17px;
     font-weight: 500;
-    text-align: center;
-    padding: 15px;
     width: 200px;
     color: #172b4d;
     margin-bottom: 10px;
     cursor: pointer;
     transition: 0.3;
-    margin-left: 20px;
   }
 
-  .btn:hover {
-    background-color: #5e6c84;
+  /* Works Section */
+
+  .works-section {
+    display: grid;
+    grid-template-rows: repeat(4);
+    padding: 200 24px;
+    gap: 86px;
+  }
```

Comment:
The design needs to breathe 🧘🏽  in order to allow your design to do so, make sure to adjust the margiin between the work section and the section previously created. See below 👇🏽 

## Your design
![Screenshot 2023-01-13 at 08 16 45](https://user-images.githubusercontent.com/45739963/212328772-a5d7fbdf-3cbe-4fb4-be0b-901d66ab7996.png)

## Figma Design

![Screenshot 2023-01-13 at 08 17 04](https://user-images.githubusercontent.com/45739963/212328829-1b48ebbb-4ae0-436a-be24-e66adf0699a1.png)